### ISSUE TITLE
All `.icon` in `accordion-header` are rotated when accordion is open

### ISSUE THREAD

**Issue by codebling** [picturepan2/spectre#625]
An accordion with an icon in the title looks ok when collapsed. 
![image](https://user-images.githubusercontent.com/1073841/77024720-a7707580-6965-11ea-86b2-ddbafc6e6604.png)

When it's open, the icons in the title are rotated. 
![image](https://user-images.githubusercontent.com/1073841/77024747-bc4d0900-6965-11ea-9a51-23a3cdb9ccea.png)

Is there a way around this? 

The following code sample should produce the example above. 
```
<div class="accordion">
  <input type="checkbox" id="accordion-1" name="accordion-checkbox" hidden />
  <label class="accordion-header" for="accordion-1">
    <i class="icon icon-arrow-right mr-1"></i>
    This is an accordion. 
    <i class="icon icon-emoji"></i>
  </label>
  <div class="accordion-body">
    <ul class="menu menu-nav">
      <li class="menu-item">hello</li>
    </ul>
  </div>
</div>
```

**Comment by Whateverinc** [comment]
I used the following code. 

`.accordion input:checked~.accordion-header .icon+.icon, .accordion[open] .accordion-header .icon+.icon {
    transform: rotate(0);
}`

**Comment by Whateverinc** [comment]
I believe that in the future, something like this should be added instead. 

`.accordion input:checked~.accordion-header .icon:first-child, .accordion[open] .accordion-header .icon:first-child {
    transform: rotate(90deg);
}`

**Comment by codebling** [comment]
Thanks for looking. Looks very similar to what I did in [the PR](https://github.com/picturepan2/spectre/pull/626/commits/1e6d5e570e277cbbfcdb0cda8989c351110ff8f2), although it seems like there are some spaces in there that shouldn't be

**Comment by picturepan2** [comment]
Fix merged in v0.5.9. Thank you.
